### PR TITLE
Add filename.png to gitignore

### PR DESCRIPTION
**Pull request by iofall**
## Description
`filename.png` gets generated every time we run `plot_quickstart.py`. On the last line we save a figure which leads to the file being saved in the `examples` directory.

## Tests

- [x] no new tests required

## Documentation

- [x] example notebook added or updated

## Changes
- Added `filename.png` to gitignore.

### CODE REVIEW

File: examples/plot_quickstart.py [hunk 1]
```diff
@@ -92,3 +92,8 @@
     title="Show count metric in pie chart",
 )
 fig[0][0].figure.savefig("filename.png")
+
+# sphinx_gallery_start_ignore
+import os
```
Comment (romanlutz) [inline examples/plot_quickstart.py:97]: Instead of this code in the notebook, perhaps adding an exception for 'filename.png' in `.gitignore` is simpler?

Comment (iofall) [reply examples/plot_quickstart.py:97]: Yep, I think that will be better.

File: .gitignore [hunk 2]
```diff
@@ -118,3 +118,6 @@ docbuild/
 
 # OSX directory info files
 .DS_Store
+
+# plot_quickstart.py
+filename.png
```
Comment (romanlutz) [inline .gitignore:123]: nit: I believe it gets generated under `examples` so perhaps `**/filename.png` helps to account for such files.

**Review by romanlutz — APPROVED** [review]
LGTM! @fairlearn/fairlearn-maintainers please review and merge if you agree, or provide feedback 🙂 Thanks @iofall !

**Comment by adrinjalali** [comment]
It would have been much nicer to fix the tests so that it doesn't produce the file :D

**Pull request merged**